src/python/pants_release/generate_github_workflows.py

```diff
@@ -913,6 +919,32 @@ def build_wheels_job(
                 helper.upload_log_artifacts(name="wheels-and-pex"),
                 *(
                     [
+                        {
+                            "name": "Attest the pantsbuild.pants wheel",
+                            #"if": "needs.release_info.outputs.is-release == 'true'",
+                            "uses": action("attest-build-provenance"),
+                            "with": {
+                                "subject-path": "dist/deploy/wheels/pantsbuild.pants/**/pantsbuild.pants-*.whl",
+                            },
+                        },
+                        {
+                            "name": "Rename the Pex file for attestation",
```

Comment:
We're renaming not just for attestation, right? We do so because we upload to the target name. So this description is a little misleading perhaps.

Reply:
Right, the rename is for the upload as well, but I had to refactor it out of the upload step so we attest the final filename.

Reply:
So yeah, I rename the step to be more clear.